### PR TITLE
Added GitHub workflows

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
This PR depends on https://github.com/th2-net/.github/pull/71

### CODE REVIEW

File: .github/workflows/build-pre-release.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    uses: th2-net/.github/.github/workflows/license_check.yml@main
+    needs: [ prebuild-job ]
+    with:
+      version: ${{ needs.prebuild-job.outputs.version }}
+  owasp-scan:
+    uses: th2-net/.github/.github/workflows/owasp-gradle-scan.yml@main
+    with:
+      multiproject: false
+      javaVersion: 11
+      javaVendor: 'zulu'
```
Comment (OptimumCode) [inline .github/workflows/build-pre-release.yml:19]: suggestion: to reduce duplication you can extract the repeating values into env variables

```yaml
env:
  JVM_VERSION: 11
  JVM_VENDOR: 'zulu'
jobs:
  owasp-scan:
    with:
      javaVersion: ${{ env.JVM_VERSION }}
```

File: .github/workflows/build-pre-release.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ needs.prebuild-job.outputs.version }}
+          release_name: Release ${{ needs.prebuild-job.outputs.version }}
+          draft: false
+          prerelease: true
+      - name: Upload Release Asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (OptimumCode) [inline .github/workflows/build-pre-release.yml:62]: note: I am not sure but you probably need to specify permission for job to allow publishing release assets

Comment (Nikita-Smirnov-Exactpro) [reply .github/workflows/build-pre-release.yml:62]: Let's try to run this workflow

Comment (Nikita-Smirnov-Exactpro) [reply .github/workflows/build-pre-release.yml:65]: No, it will not work

Comment (OptimumCode) [reply .github/workflows/build-pre-release.yml:65]: Okay, but I am more than sure that we will have to specify write permissions here explicitly or provide a token with write access to releases

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**